### PR TITLE
bpo-30619: fix

### PR DESCRIPTION
**Pull request by khyox**
Following [bug #30619](http://bugs.python.org/issue30619).

It was the example (and not the text) which is consistent with PEP 483:

    Corollary: Union[..., object, ...] returns object

So, the correction is simply to substitute 'former' for 'latter'.

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately we couldn't find an account corresponding to your GitHub username on [bugs.python.org](http://bugs.python.org/) (b.p.o) to verify you have signed the CLA (this might be simply due to a missing "GitHub Name" entry in your b.p.o account settings). This is necessary for legal reasons before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://cpython-devguide.readthedocs.io/pullrequest.html#licensing) to rectify this issue.


Thanks again to your contribution and we look forward to looking at it!

**Comment by mlouielu** [comment]
@khyox Hello and welcome, you will need to reform your title to start with `bpo-30619: <your title>`, so that the bedevere may get the correct bpo issue number!

**Review by ilevkivskyi — APPROVED** [review]
LGTM, Thanks!

I hope someone with commit rights will merge this after you sign the CLA.

**Comment by khyox** [comment]
Thanks @mlouielu and @ilevkivskyi ! CLA just signed.

**Review by Mariatta — APPROVED** [review]

**Comment by Mariatta** [comment]
Thanks @khyox and congrats on your first contribution to CPython :tada:

**Comment by khyox** [comment]
Thank you very much @Mariatta!! Just a tiny bit, but it's a honor indeed! I hope to keep contributing in the near future.

**Pull request merged**
